### PR TITLE
Updating the Dockerfile to take into account the latest changes to the code base

### PR DESCRIPTION
**Pull request by danakim**

### CODE REVIEW

File: contrib/docker/Dockerfile [hunk 1]
```diff
@@ -12,13 +13,13 @@ RUN apt-get -y install curl git vim
...
+VOLUME /var/lib/subtext
 RUN cd /opt && git clone https://github.com/mgax/subtext.git
 WORKDIR /opt/subtext
 RUN npm install
+RUN ./run build
 
 # Start subtext
-ENTRYPOINT ["./run", "server", "${path_to_identity}"]
+ENV AUTH_TOKEN ${auth_token}
+ENTRYPOINT ["./run", "server", "/var/lib/subtext", "http://subtext.hostatic.ro"]
```
Comment (mgax) [inline contrib/docker/Dockerfile]: Should we be hardcoding `subtext.hostatic.ro`? :)

Comment (danakim) [inline contrib/docker/Dockerfile]: /facepalm - forgot that one in the commit. Oops :D

File: contrib/docker/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,5 @@
 FROM ubuntu:14.04
```
Comment (mgax) [inline contrib/docker/Dockerfile:1]: Does it have to be based on ubuntu? I think debian jessie is lighter.

Comment (danakim) [inline contrib/docker/Dockerfile:1]: I had some trouble getting Node to run on debian last time I tried, so I defaulted to what I knew that worked well.

I will make a note to change it to something else, like Debian, before we upload it to Docker hub.

Comment (mgax) [inline contrib/docker/Dockerfile:1]: > I had some trouble getting Node to run on debian

I'm using Debian with the nodesource dpkg server with no issues. But well, it's not with docker.

Comment (danakim) [reply contrib/docker/Dockerfile:1]: Just tried the Debian image: the difference is not that big - the Ubuntu one has 180 Mb, the Debian one 130Mb. But the Debian one has a lot of stuff missing, including sudo :) and the proper apt sources files. So getting it to work is a bit of a PITA, while on Ubuntu it 'just worked'. So I would say we leave it on Ubuntu for now, as the time wasted is not worth the 50Mbs. And you know what they say, premature optimisation is the root of all evil ;)

Comment (mgax) [reply contrib/docker/Dockerfile:1]: > the time wasted is not worth the 50Mbs

I totally agree. Thanks for the PR! :cake:

**Comment by danakim** [comment]
Glad I could convince you! And yay, PR merged \o/

Cu mare placere ;)

**Pull request merged**